### ISSUE TITLE
False negative on insufficient arguments with `*args: *tuple[?]` since 1.1.382

### ISSUE THREAD

**Issue by Azureblade3808** [microsoft/pyright#9183]
**Describe the bug**

When a function has variadic arguments with type of unpacking a tuple, calling it with insufficient argument(s) incorrectly passes type checking.

It used to work with Pyright 1.1.381.

**Code or Screenshots**

```python
def f(*args: *tuple[int, str]): ...

f()  # Expected to be reported, but not.
f(0)  # Expected to be reported, but not.
f(0, "")  # Passes.
```

**VS Code extension or command-line**

Pyright 1.1.382 & 1.1.383

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.385.
